include/dronecore.h

```diff
@@ -10,12 +10,41 @@ namespace dronecore {
 
 class DroneCoreImpl;
 
+/**
+ * DroneCore is an API and library for the PX4 flight stack using Mavlink.
```

Comment:
Later on we should revisit this text. All correct, but not the full story.

Reply:
@julianoes Note this comment, requires action.

Reply:
I did change it a bit:
https://github.com/dronecore/DroneCore/blob/131deb4bd7ab2e2eb47f38c167c2f020e237655d/include/dronecore.h#L14-L17
Still no good?